bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormStructureParser.java

```diff
@@ -43,4 +43,10 @@ public interface FormStructureParser {
      * @return true if this resource or one of its children is a form container, else false
      */
     Boolean containsFormContainer();
+
+    /**
+     *
+     * @return form definition json in Publish view
+     */
+    String getFormDefinition();
```

Comment:
Please bump the minor version and add a @since tag in javadoc mentioning the new version. You can check the existing code for reference